images/iptables/Dockerfile

```
@@ -0,0 +1,21 @@
+# Copyright Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+ARG IPTABLES_VERSION=1.8.8-1
+
+FROM ubuntu:22.04
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends debian-archive-keyring apt-src
+
+RUN echo 'deb-src [signed-by=/usr/share/keyrings/debian-archive-bullseye-automatic.gpg] http://deb.debian.org/debian bookworm main' > /etc/apt/sources.list.d/debian-testing.list && \
+    apt-get update
+
+RUN mkdir /iptables
+WORKDIR /iptables
+
+ARG IPTABLES_VERSION
+RUN apt-src -b install iptables=${IPTABLES_VERSION}
+
+RUN apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
```

How about merging all the `RUN` steps involving `apt-{get,src}`?

```suggestion
RUN mkdir /iptables
WORKDIR /iptables

ARG IPTABLES_VERSION

RUN echo 'deb-src [signed-by=/usr/share/keyrings/debian-archive-bullseye-automatic.gpg] http://deb.debian.org/debian bookworm main' > /etc/apt/sources.list.d/debian-testing.list && \
    apt-get update && \
    apt-src -b install iptables=${IPTABLES_VERSION} && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```

Would that please the linter?